.vscode/settings.json

```diff
@@ -83,5 +83,6 @@
   "python.testing.unittestEnabled": false,
   "python.testing.nosetestsEnabled": false,
   "python.testing.pytestEnabled": true,
-  "python.testing.autoTestDiscoverOnSaveEnabled": true
+  "python.testing.autoTestDiscoverOnSaveEnabled": true,
+  "jupyter.jupyterServerType": "local"
```

Comment:
this probably personal preference, some of dev may use remote server.
Do you mind move this to your user setting instead of check in to repo?

Reply:
Yeah sorry, missed that it changed for repo